src/components/App.js

```diff
@@ -1,8 +1,196 @@
+import React, { useState } from 'react';
+import SelectNumber from './Select';
+import Board from './Board';
+import { alphabet } from '../letters/Alphabet';
+import { wordDictionary2 } from '../letters/WordDictionary2';
+import { wordDictionary3 } from '../letters/WordDictionary3';
+import { wordDictionary4 } from '../letters/WordDictionary4';
+import { wordDictionary5 } from '../letters/WordDictionary5';
+import { letterPoints } from '../letters/LetterPoints';
 
 function App() {
+  const [selectNumber, setSelectNumber] = useState(false);
+  const [wordNo, setWordNo] = useState(1);
+  const [numberSelected, setNumberSelected] = useState('');
+  const [squares, setSquares] = useState([]);
+  const [wordLengths, setWordLengths] = useState([]);
+  const [remainingAlphabet, setRemainingAlphabet] = useState(alphabet);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [score, setScore] = useState(0);
+  let game;
+  let words = [];
+  let invalidWord = '';
+
+  class Game {
```

Comment:
hrmmm this is definitely not what I was expecting when I asked for a class 😅 
Maybe a class is the wrong direction, we will eventually get to the problem of how do we represent a class instance in state, that gets tricky. Maybe a custom hook is the better approach?

Could you make a `useBoard` custom hook. It can simply just contain your squares state and reset method and such

Reply:
I think it would also help if all this board-y stuff moved into the `Board` component so it doesn't get all mixed up with the configuration settings (e.g. selectNumber/numberSelected)

Reply:
The class structure has been removed.